sycl-jit/jit-compiler/lib/rtc/DeviceCompilation.cpp

```diff
@@ -51,6 +51,13 @@
 #include <array>
 #include <sstream>
 
+namespace jit_compiler {
+// Defined in the auto-generated file:
+extern const std::pair<std::string_view, std::string_view> ToolchainFiles[];
```

Comment:
Are there any ABI problems looming here?

Reply:
I wouldn't think so - the use is limited to `libsycl-jit.so` and this isn't being exported outside the DSO.

Reply:
> the TU containing ToolchainFiles is compiled with a different compiler and flags than the rest of sycl-jit.so, so maybe there's future ABI problem hiding in there.

I see now what you meant. I believe if any such issues will happen, we'll catch them immediately with our E2E tests, because now that these files are distributed with the `libsycl-jit.so` we have much more reproducibility in the behavior. Also, `resource.cpp.o` is very simple after compiling with optimizations.